libycp/src/include/ycp/Bytecode.h

```diff
@@ -38,6 +38,9 @@ class Y2Namespace;
 
 #include <fstream>
 
+#undef major
+#undef minor
```

Comment:
why it is needed? I think we define it only in bytecodeistream

Reply:
See the warning quoted in the commit message

Reply:
yes, but it mention ` If you did not intend to use a system-defined macro
  "major", you should undefine it after including <sys/types.h>.` but I do not see any sys/types.h here. And for me commit message is not enough, we should have it in comment above strange code.

Reply:
so what I want to say here is that it is not straightforward why it is needed. and digging into commits for why is not perfect way for me. Of course explanation can be a bit shorter then in commit message.

Reply:
comment added. better now?